sql/opt/exec/execbuilder: TestExecBuild failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/e6359159252597cab72855673bcd9334d339c0f9

Parameters:
```
TAGS=
GOFLAGS=-parallel=4
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestExecBuild PKG=github.com/cockroachdb/cockroach/pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1304288&tab=buildLog

```
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/9/#/62/1/1{-/#}
                        querying next range at /Table/61/1/9/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/9/#/62/1/1/0
                        querying next range at /Table/61/1/9/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        fetched: /a/primary/10 -> NULL
                        Del /Table/61/1/10/0
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/10/#/62/1/1{-/#}
                        querying next range at /Table/61/1/10/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/10/#/62/1/1/0
                        querying next range at /Table/61/1/10/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 10 Del to (n1,s1):1
                        output row: [1]
                        output row: [2]
                        output row: [3]
                        output row: [4]
                        output row: [5]
                        output row: [6]
                        output row: [7]
                        output row: [8]
                        output row: [9]
                        output row: [10]
                        rows affected: 10
                        querying next range at /Table/61/1/1/#/62/1/1/0
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 1 EndTxn to (n1,s1):1
                        r20: sending batch 10 QueryIntent to (n1,s1):1
                        
                    
                logic.go:2314: 
                     testdata/interleaved:575: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local-opt/interleaved
=== PAUSE TestExecBuild/local-opt/interleaved
=== CONT  TestExecBuild/local-opt/interleaved
--- done: testdata/delete with config local-opt: 21 tests, 0 failures
        --- FAIL: TestExecBuild/local-opt/interleaved (1.09s)
=== RUN   TestExecBuild/local-opt
    --- FAIL: TestExecBuild/local-opt (0.00s)

```

SHA: https://github.com/cockroachdb/cockroach/commits/e6359159252597cab72855673bcd9334d339c0f9

Parameters:
```
TAGS=
GOFLAGS=-race -parallel=2
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestExecBuild PKG=github.com/cockroachdb/cockroach/pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1304290&tab=buildLog

```
                        querying next range at /Table/61/1/8/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        fetched: /a/primary/9 -> NULL
                        Del /Table/61/1/9/0
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/9/#/62/1/1{-/#}
                        querying next range at /Table/61/1/9/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/9/#/62/1/1/0
                        querying next range at /Table/61/1/9/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        fetched: /a/primary/10 -> NULL
                        Del /Table/61/1/10/0
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/10/#/62/1/1{-/#}
                        querying next range at /Table/61/1/10/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/10/#/62/1/1/0
                        querying next range at /Table/61/1/10/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 10 Del to (n1,s1):1
                        output row: [1]
                        output row: [2]
                        output row: [3]
                        output row: [4]
                        output row: [5]
                        output row: [6]
                        output row: [7]
                        output row: [8]
                        output row: [9]
                        output row: [10]
                        rows affected: 10
                        querying next range at /Table/61/1/1/#/62/1/1/0
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 1 EndTxn to (n1,s1):1
                        r20: sending batch 10 QueryIntent to (n1,s1):1
                        
                    
                logic.go:2314: 
                     testdata/interleaved:575: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local-opt/interleaved
        --- FAIL: TestExecBuild/local-opt/interleaved (2.88s)

```

SHA: https://github.com/cockroachdb/cockroach/commits/630a6e9cb3771912cd138f9aa3bea1f0ca9fa7c9

Parameters:
```
TAGS=
GOFLAGS=-parallel=4
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestExecBuild PKG=github.com/cockroachdb/cockroach/pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1306565&tab=buildLog

```
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/9/#/62/1/1{-/#}
                        querying next range at /Table/61/1/9/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/9/#/62/1/1/0
                        querying next range at /Table/61/1/9/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        fetched: /a/primary/10 -> NULL
                        Del /Table/61/1/10/0
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/10/#/62/1/1{-/#}
                        querying next range at /Table/61/1/10/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/10/#/62/1/1/0
                        querying next range at /Table/61/1/10/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 10 Del to (n1,s1):1
                        output row: [1]
                        output row: [2]
                        output row: [3]
                        output row: [4]
                        output row: [5]
                        output row: [6]
                        output row: [7]
                        output row: [8]
                        output row: [9]
                        output row: [10]
                        rows affected: 10
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 10 QueryIntent to (n1,s1):1
                        querying next range at /Table/61/1/1/#/62/1/1/0
                        r20: sending batch 1 EndTxn to (n1,s1):1
                        
                    
                logic.go:2314: 
                     testdata/interleaved:575: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local-opt/interleaved
=== PAUSE TestExecBuild/local-opt/interleaved
=== CONT  TestExecBuild/local-opt/interleaved
--- done: testdata/inverted_index with config local-opt: 26 tests, 0 failures
--- total progress: 683 statements/queries
        --- FAIL: TestExecBuild/local-opt/interleaved (0.86s)

```

SHA: https://github.com/cockroachdb/cockroach/commits/630a6e9cb3771912cd138f9aa3bea1f0ca9fa7c9

Parameters:
```
TAGS=
GOFLAGS=-race -parallel=2
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestExecBuild PKG=github.com/cockroachdb/cockroach/pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1306566&tab=buildLog

```
                        fetched: /a/primary/9 -> NULL
                        Del /Table/61/1/9/0
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/9/#/62/1/1{-/#}
                        querying next range at /Table/61/1/9/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/9/#/62/1/1/0
                        querying next range at /Table/61/1/9/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/9/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/9/#/62/{1-2}
                        querying next range at /Table/61/1/9/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        fetched: /a/primary/10 -> NULL
                        Del /Table/61/1/10/0
                        cascading delete into table: 62 using index: 1
                        CascadeScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        CascadeScan /Table/61/1/10/#/62/1/1{-/#}
                        querying next range at /Table/61/1/10/#/62/1/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        Del /Table/61/1/10/#/62/1/1/0
                        querying next range at /Table/61/1/10/#/62/1/1/0
                        r20: sending batch 1 Del to (n1,s1):1
                        cascading delete into table: 63 using index: 1
                        CascadeScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
                        querying next range at /Table/61/1/10/#/62/1/1/#/63/1
                        r20: sending batch 1 Scan to (n1,s1):1
                        FKScan /Table/61/1/10/#/62/{1-2}
                        querying next range at /Table/61/1/10/#/62/1
                        r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 10 Del to (n1,s1):1
                        output row: [1]
                        output row: [2]
                        output row: [3]
                        output row: [4]
                        output row: [5]
                        output row: [6]
                        output row: [7]
                        output row: [8]
                        output row: [9]
                        output row: [10]
                        rows affected: 10
                        querying next range at /Table/61/1/1/#/62/1/1/0
                        querying next range at /Table/61/1/1/0
                        r20: sending batch 1 EndTxn to (n1,s1):1
                        r20: sending batch 10 QueryIntent to (n1,s1):1
                        
                    
                logic.go:2314: 
                     testdata/interleaved:575: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local-opt/interleaved
        --- FAIL: TestExecBuild/local-opt/interleaved (2.51s)
=== RUN   TestExecBuild/local-opt
    --- FAIL: TestExecBuild/local-opt (95.41s)

```

--- a/pkg/sql/logictest/testdata/planner_test/interleaved
+++ b/pkg/sql/logictest/testdata/planner_test/interleaved
@@ -238,16 +238,14 @@ c_id  a_id  b_id
 statement ok
 SET TRACING = on,kv,results; DELETE FROM a where a_id <= 7 and a_id >= 5; SET tracing = off
 
+# Only look at traces from SQL land.
 query T
 SELECT message FROM [SHOW KV TRACE FOR SESSION]
+WHERE operation='flow' OR operation LIKE 'exec cmd:%'
 ----
 DelRange /Table/61/1/5 - /Table/61/1/7/NULL
-querying next range at /Table/61/1/5
-r20: sending batch 1 DelRng to (n1,s1):1
 fast path completed
 rows affected: 3
-querying next range at /Table/61/1/5
-r20: sending batch 1 EndTxn to (n1,s1):1
 
 query II colnames
 select * from b order by a_id, b_id
@@ -283,15 +281,12 @@ statement ok
 SET TRACING = on,kv,results; DELETE FROM a; SET tracing = off
 
 query T
-select message FROM [SHOW KV TRACE FOR SESSION]
+SELECT message FROM [SHOW KV TRACE FOR SESSION]
+WHERE operation='flow' OR operation LIKE 'exec cmd:%'
 ----
 DelRange /Table/61/1 - /Table/61/3
-querying next range at /Table/61/1
-r20: sending batch 1 DelRng to (n1,s1):1
 fast path completed
 rows affected: 5
-querying next range at /Table/61/1
-r20: sending batch 1 EndTxn to (n1,s1):1
 
 query II colnames
 select * from b order by a_id, b_id
@@ -328,233 +323,99 @@ statement ok
 SET TRACING=off;
 
 query T
-select message FROM [SHOW KV TRACE FOR SESSION];
+SELECT message FROM [SHOW KV TRACE FOR SESSION]
+WHERE operation IN ('flow', 'consuming rows') OR operation LIKE 'exec cmd:%'
 ----
-Scan /Table/61/{1-2}
-querying next range at /Table/61/1
-r20: sending batch 1 Scan to (n1,s1):1
-fetched: /a/primary/1 -> NULL
 Del /Table/61/1/1/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/1/#/62/{1-2}
-querying next range at /Table/61/1/1/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/1/#/62/1/1{-/#}
-querying next range at /Table/61/1/1/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/1/#/62/1/1/0
-querying next range at /Table/61/1/1/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/1/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/1/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/1/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/1/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/1/#/62/{1-2}
-querying next range at /Table/61/1/1/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/2 -> NULL
 Del /Table/61/1/2/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/2/#/62/{1-2}
-querying next range at /Table/61/1/2/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/2/#/62/1/1{-/#}
-querying next range at /Table/61/1/2/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/2/#/62/1/1/0
-querying next range at /Table/61/1/2/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/2/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/2/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/2/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/2/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/2/#/62/{1-2}
-querying next range at /Table/61/1/2/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/3 -> NULL
 Del /Table/61/1/3/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/3/#/62/{1-2}
-querying next range at /Table/61/1/3/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/3/#/62/1/1{-/#}
-querying next range at /Table/61/1/3/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/3/#/62/1/1/0
-querying next range at /Table/61/1/3/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/3/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/3/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/3/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/3/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/3/#/62/{1-2}
-querying next range at /Table/61/1/3/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/4 -> NULL
 Del /Table/61/1/4/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/4/#/62/{1-2}
-querying next range at /Table/61/1/4/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/4/#/62/1/1{-/#}
-querying next range at /Table/61/1/4/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/4/#/62/1/1/0
-querying next range at /Table/61/1/4/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/4/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/4/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/4/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/4/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/4/#/62/{1-2}
-querying next range at /Table/61/1/4/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/5 -> NULL
 Del /Table/61/1/5/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/5/#/62/{1-2}
-querying next range at /Table/61/1/5/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/5/#/62/1/1{-/#}
-querying next range at /Table/61/1/5/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/5/#/62/1/1/0
-querying next range at /Table/61/1/5/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/5/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/5/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/5/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/5/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/5/#/62/{1-2}
-querying next range at /Table/61/1/5/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/6 -> NULL
 Del /Table/61/1/6/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/6/#/62/{1-2}
-querying next range at /Table/61/1/6/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/6/#/62/1/1{-/#}
-querying next range at /Table/61/1/6/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/6/#/62/1/1/0
-querying next range at /Table/61/1/6/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/6/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/6/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/6/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/6/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/6/#/62/{1-2}
-querying next range at /Table/61/1/6/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/7 -> NULL
 Del /Table/61/1/7/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/7/#/62/{1-2}
-querying next range at /Table/61/1/7/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/7/#/62/1/1{-/#}
-querying next range at /Table/61/1/7/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/7/#/62/1/1/0
-querying next range at /Table/61/1/7/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/7/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/7/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/7/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/7/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/7/#/62/{1-2}
-querying next range at /Table/61/1/7/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/8 -> NULL
 Del /Table/61/1/8/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/8/#/62/{1-2}
-querying next range at /Table/61/1/8/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/8/#/62/1/1{-/#}
-querying next range at /Table/61/1/8/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/8/#/62/1/1/0
-querying next range at /Table/61/1/8/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/8/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/8/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/8/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/8/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/8/#/62/{1-2}
-querying next range at /Table/61/1/8/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/9 -> NULL
 Del /Table/61/1/9/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/9/#/62/{1-2}
-querying next range at /Table/61/1/9/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/9/#/62/1/1{-/#}
-querying next range at /Table/61/1/9/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/9/#/62/1/1/0
-querying next range at /Table/61/1/9/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/9/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/9/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/9/#/62/{1-2}
-querying next range at /Table/61/1/9/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/10 -> NULL
 Del /Table/61/1/10/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/10/#/62/{1-2}
-querying next range at /Table/61/1/10/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/10/#/62/1/1{-/#}
-querying next range at /Table/61/1/10/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/10/#/62/1/1/0
-querying next range at /Table/61/1/10/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/10/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/10/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/10/#/62/{1-2}
-querying next range at /Table/61/1/10/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-querying next range at /Table/61/1/1/0
-r20: sending batch 10 Del, 1 EndTxn to (n1,s1):1
 output row: [1]
 output row: [2]
 output row: [3]

--- a/pkg/sql/opt/exec/execbuilder/testdata/interleaved
+++ b/pkg/sql/opt/exec/execbuilder/testdata/interleaved
@@ -238,16 +238,14 @@ c_id  a_id  b_id
 statement ok
 SET TRACING = on,kv,results; DELETE FROM a where a_id <= 7 and a_id >= 5; SET tracing = off
 
+# Only look at traces from SQL land.
 query T
 SELECT message FROM [SHOW KV TRACE FOR SESSION]
+WHERE operation='flow' OR operation LIKE 'exec cmd:%'
 ----
 DelRange /Table/61/1/5 - /Table/61/1/7/NULL
-querying next range at /Table/61/1/5
-r20: sending batch 1 DelRng to (n1,s1):1
 fast path completed
 rows affected: 3
-querying next range at /Table/61/1/5
-r20: sending batch 1 EndTxn to (n1,s1):1
 
 query II colnames
 select * from b order by a_id, b_id
@@ -283,15 +281,12 @@ statement ok
 SET TRACING = on,kv,results; DELETE FROM a; SET tracing = off
 
 query T
-select message FROM [SHOW KV TRACE FOR SESSION]
+SELECT message FROM [SHOW KV TRACE FOR SESSION]
+WHERE operation='flow' OR operation LIKE 'exec cmd:%'
 ----
 DelRange /Table/61/1 - /Table/61/3
-querying next range at /Table/61/1
-r20: sending batch 1 DelRng to (n1,s1):1
 fast path completed
 rows affected: 5
-querying next range at /Table/61/1
-r20: sending batch 1 EndTxn to (n1,s1):1
 
 query II colnames
 select * from b order by a_id, b_id
@@ -328,233 +323,99 @@ statement ok
 SET TRACING=off;
 
 query T
-select message FROM [SHOW KV TRACE FOR SESSION];
+SELECT message FROM [SHOW KV TRACE FOR SESSION]
+WHERE operation IN ('flow', 'consuming rows') OR operation LIKE 'exec cmd:%'
 ----
-Scan /Table/61/{1-2}
-querying next range at /Table/61/1
-r20: sending batch 1 Scan to (n1,s1):1
-fetched: /a/primary/1 -> NULL
 Del /Table/61/1/1/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/1/#/62/{1-2}
-querying next range at /Table/61/1/1/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/1/#/62/1/1{-/#}
-querying next range at /Table/61/1/1/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/1/#/62/1/1/0
-querying next range at /Table/61/1/1/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/1/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/1/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/1/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/1/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/1/#/62/{1-2}
-querying next range at /Table/61/1/1/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/2 -> NULL
 Del /Table/61/1/2/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/2/#/62/{1-2}
-querying next range at /Table/61/1/2/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/2/#/62/1/1{-/#}
-querying next range at /Table/61/1/2/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/2/#/62/1/1/0
-querying next range at /Table/61/1/2/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/2/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/2/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/2/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/2/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/2/#/62/{1-2}
-querying next range at /Table/61/1/2/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/3 -> NULL
 Del /Table/61/1/3/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/3/#/62/{1-2}
-querying next range at /Table/61/1/3/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/3/#/62/1/1{-/#}
-querying next range at /Table/61/1/3/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/3/#/62/1/1/0
-querying next range at /Table/61/1/3/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/3/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/3/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/3/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/3/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/3/#/62/{1-2}
-querying next range at /Table/61/1/3/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/4 -> NULL
 Del /Table/61/1/4/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/4/#/62/{1-2}
-querying next range at /Table/61/1/4/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/4/#/62/1/1{-/#}
-querying next range at /Table/61/1/4/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/4/#/62/1/1/0
-querying next range at /Table/61/1/4/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/4/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/4/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/4/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/4/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/4/#/62/{1-2}
-querying next range at /Table/61/1/4/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/5 -> NULL
 Del /Table/61/1/5/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/5/#/62/{1-2}
-querying next range at /Table/61/1/5/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/5/#/62/1/1{-/#}
-querying next range at /Table/61/1/5/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/5/#/62/1/1/0
-querying next range at /Table/61/1/5/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/5/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/5/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/5/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/5/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/5/#/62/{1-2}
-querying next range at /Table/61/1/5/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/6 -> NULL
 Del /Table/61/1/6/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/6/#/62/{1-2}
-querying next range at /Table/61/1/6/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/6/#/62/1/1{-/#}
-querying next range at /Table/61/1/6/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/6/#/62/1/1/0
-querying next range at /Table/61/1/6/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/6/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/6/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/6/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/6/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/6/#/62/{1-2}
-querying next range at /Table/61/1/6/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/7 -> NULL
 Del /Table/61/1/7/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/7/#/62/{1-2}
-querying next range at /Table/61/1/7/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/7/#/62/1/1{-/#}
-querying next range at /Table/61/1/7/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/7/#/62/1/1/0
-querying next range at /Table/61/1/7/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/7/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/7/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/7/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/7/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/7/#/62/{1-2}
-querying next range at /Table/61/1/7/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/8 -> NULL
 Del /Table/61/1/8/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/8/#/62/{1-2}
-querying next range at /Table/61/1/8/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/8/#/62/1/1{-/#}
-querying next range at /Table/61/1/8/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/8/#/62/1/1/0
-querying next range at /Table/61/1/8/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/8/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/8/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/8/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/8/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/8/#/62/{1-2}
-querying next range at /Table/61/1/8/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/9 -> NULL
 Del /Table/61/1/9/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/9/#/62/{1-2}
-querying next range at /Table/61/1/9/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/9/#/62/1/1{-/#}
-querying next range at /Table/61/1/9/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/9/#/62/1/1/0
-querying next range at /Table/61/1/9/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/9/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/9/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/9/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/9/#/62/{1-2}
-querying next range at /Table/61/1/9/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-fetched: /a/primary/10 -> NULL
 Del /Table/61/1/10/0
 cascading delete into table: 62 using index: 1
 CascadeScan /Table/61/1/10/#/62/{1-2}
-querying next range at /Table/61/1/10/#/62/1
-r20: sending batch 1 Scan to (n1,s1):1
 CascadeScan /Table/61/1/10/#/62/1/1{-/#}
-querying next range at /Table/61/1/10/#/62/1/1
-r20: sending batch 1 Scan to (n1,s1):1
 Del /Table/61/1/10/#/62/1/1/0
-querying next range at /Table/61/1/10/#/62/1/1/0
-r20: sending batch 1 Del to (n1,s1):1
 cascading delete into table: 63 using index: 1
 CascadeScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/10/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/10/#/62/1/1/#/63/{1-2}
-querying next range at /Table/61/1/10/#/62/1/1/#/63/1
-r20: sending batch 1 Scan to (n1,s1):1
 FKScan /Table/61/1/10/#/62/{1-2}
-querying next range at /Table/61/1/10/#/62/1
-r20: sending batch 1 Scan, 1 QueryIntent to (n1,s1):1
-querying next range at /Table/61/1/1/0
-r20: sending batch 10 Del to (n1,s1):1
 output row: [1]
 output row: [2]
 output row: [3]
@@ -566,10 +427,6 @@ output row: [8]
 output row: [9]
 output row: [10]
 rows affected: 10
-querying next range at /Table/61/1/1/#/62/1/1/0
-r20: sending batch 1 EndTxn to (n1,s1):1
-querying next range at /Table/61/1/1/0
-r20: sending batch 10 QueryIntent to (n1,s1):1
 
 statement ok
 DROP TABLE c; DROP TABLE b; DROP TABLE a

